src/runtimes/node/parser/exports.ts

```diff
@@ -74,16 +101,20 @@ const getMainExportFromCJS = (node: Statement) => {
   })
 }
 
-// Finds the main handler export in an ESM AST.
-const getMainExportFromESM = (node: Statement, getAllBindings: BindingMethod) => {
+/**
+ * Finds a named ESM export with a given name. It's capable of finding exports
+ * with a variable declaration (`export const foo = "bar"`), but also resolve
+ * bindings and find things like `const baz = "1"; export { baz as foo }`.
+ */
+const getNamedESMExport = (node: Statement, name: string, getAllBindings: BindingMethod) => {
```

Comment:
Renamed from `getMainExportFromESM`. This is now generic so it can look for exports with different names.